### DISCUSSION TITLE
Camera's MediaCaptureFailedEventArgs

### DISCUSSION THREAD

**Discussion by tschbc** [CommunityToolkit/Maui#2035] [Q&A] [answered]
The docstring for [MediaCaptureFailedEventArgs](https://github.com/CommunityToolkit/Maui/blob/main/src/CommunityToolkit.Maui.Camera/Primitives/MediaCaptureFailedEventArgs.shared.cs) says it contains error info, but checking the code itself...

```cs
﻿namespace CommunityToolkit.Maui.Views;

/// <summary>
/// Event args containing all contextual information related to a media capture failure event.
/// </summary>
public class MediaCaptureFailedEventArgs : EventArgs
{
}
```

... it's looking pretty bare bones. Are there any plans to flesh this out?

**Comment by bijington** [comment]
Good spot! Yes this really should have some details inside it

**Comment by tschbc** [comment, marked as answer]
Just saw https://github.com/CommunityToolkit/Maui/pull/2085, thank you @bijington!
